.env-prod

```diff
@@ -21,7 +21,7 @@ SCHEDULER_API=http://scheduler:8000
 KEIKO_API=http://keiko:8000
 KATALOGUS_API=http://katalogus:8000
 XTDB_URI=http://crux:3000
-BOEFJE_API=http://boefje:8000
+BOEFJES_API=http://boefje:8000
```

Comment:
We cant just change this env-var like that, as all users of previous versions will have this name already set to BOEFJE_API

Reply:
Doesn't that require and update to the installation/updating scripts too?

Reply:
yup, and hopefully some migration / deprecation warnings

Reply:
The name was already changed from BOEFJE_API to BOEFJES_API a long time ago with included backwards compatibility. This just updates our own usage to use the BOEFJES_API instead of the deprecrated BOEFJE_API.